plugins/pdr_deterministic_plugin/requirements.txt

```diff
@@ -0,0 +1,9 @@
+flask<=3.0.3
```

Comment:
good that you added a requirements file, maybe also change it in the Dockerfile not to have it duplicated